libsolidity/formal/BMC.cpp

```diff
@@ -694,6 +714,34 @@ pair<vector<smtutil::Expression>, vector<string>> BMC::modelExpressions()
 
 /// Verification targets.
 
+string BMC::targetDescription(BMCVerificationTarget const& _target)
+{
+	if (
+		_target.type == VerificationTargetType::Underflow ||
+		_target.type == VerificationTargetType::Overflow
+	)
+	{
+		auto const* intType = dynamic_cast<IntegerType const*>(_target.expression->annotation().type);
+		if (!intType)
+			intType = TypeProvider::uint256();
+
+		if (_target.type == VerificationTargetType::Underflow)
+			return "Underflow (resulting value less than " + formatNumberReadable(intType->minValue()) + ")";
+		else if (_target.type == VerificationTargetType::Overflow)
+			return "Overflow (resulting value larger than " + formatNumberReadable(intType->maxValue()) + ")";
+		else
+			solAssert(false);
```

Comment:
This can be removed, and the very last `else` branch as well (line 742). I generally prefer a `switch/case` in lieu of a ton of `else/if` statement, but this is really more of a taste thing, rather than an objective improvement.

Reply:
I can remove this one, but the very last I can't, compiler complains about termination

Reply:
updated